Cargo.toml

```diff
@@ -18,6 +18,8 @@ typetag = "0.2"
 serde_yaml = "0.9"
 dyn-clone = "1.0"
 indexmap = "1.9"
+crossbeam-channel = { version = "0.5", optional = true }
+notify = { version = "=5.0.0-pre.15", optional = true }
```

Comment:
Does this specifically need to be `=5.0.0-pre.15`? Can it work with just `5.0.0` (which is [what Bevy's using](https://github.com/bevyengine/bevy/blob/3aaf746675c150982322553e847275bc6a9d66bb/crates/bevy_asset/Cargo.toml#L33))?

Reply:
I used =5.0.0-pre.15 because that's what Bevy 0.8(.1) uses.

Reply:
Makes sense!